src/Components/RosTable/ProgressScoreBar.js

```diff
@@ -2,18 +2,11 @@ import React from 'react';
 import { Progress } from '@patternfly/react-core';
 import propTypes from 'prop-types';
 
-export const ProgressScoreBar = ({ valueScore, utilizedValue, measureLocation, eleId }) =>  {
-    const colorclass = (val) =>  ({
-        5: 'green-400',
-        4: 'green-100',
-        3: 'gold-400',
-        2: 'orange-300'
-    }[val] || 'danger-200');
```

Comment:
As you removed this logic from here, please remove respective CSS classes from `ros-page.scss` as well if not used anywhere else.

Reply:
@kgaikwad nice suggestion 👍🏻 thanks you!
I checked the code and `green-400` is being using in entitiesReducer, I removed rest of the css classes 💯